grml-debootstrap

```diff
@@ -1432,6 +1437,7 @@ prepare_vm() {
   # hopefully this always works as expected
   LOOP_PART="${DEVINFO##add map }" # 'loop0p1 (254:5): 0 20477 linear 7:0 3'
   LOOP_PART="${LOOP_PART// */}"    # 'loop0p1'
+  LOOP_DISK="${LOOP_PART%p1}"    # 'loop0'
```

Comment:
This might break in the future if we're not installing to first partition? We could consider using "%p*" or so instead?

Reply:
I would like to use "%p[0-9]" (for now) because I usually avoid "*" globbing. But if you think we ever will user double digit partitions I am willing to change that.